var/spack/repos/builtin/packages/py-pyfftw/fix_setup.patch

```diff
@@ -0,0 +1,12 @@
+--- ./setup.py.orig     2023-11-12 22:24:09.000000000 -0800
++++ ./setup.py  2023-11-12 22:26:23.000000000 -0800
+@@ -785,7 +785,8 @@
+             'Topic :: Scientific/Engineering',
+             'Topic :: Scientific/Engineering :: Mathematics',
+             'Topic :: Multimedia :: Sound/Audio :: Analysis'],
+-        'cmdclass': cmdclass
++        'cmdclass': cmdclass,
++        'packages': ['pyfftw']
```

Comment:
```suggestion
+        'packages': ['pyfftw', 'pyfftw.builders', 'pyfftw.interfaces']
```
You can see that this package list is supposed to be added below, but isn't for some reason. It looks like `pip install` runs `python setup.py egg_info` before it does anything else, which is why it is skipped.